src/indicator/algorithm/calculator/stochasticOscillator.js

```diff
@@ -0,0 +1,68 @@
+import d3 from 'd3';
+import _slidingWindow from './slidingWindow';
+import {rebind} from '../../../util/rebind';
+
+export default function() {
+
+    var closeValue = function(d, i) { return d.close; },
+        highValue = function(d, i) { return d.high; },
+        lowValue = function(d, i) { return d.low; };
+
+    var kWindow = _slidingWindow()
+        .windowSize(5)
+        .accumulator(function(values) {
+            var maxHigh = d3.max(values, highValue);
+            var minLow = d3.min(values, lowValue);
+            return 100 * (closeValue(values[values.length - 1]) - minLow) / (maxHigh - minLow);
+        });
+
+    var dWindow = _slidingWindow()
+        .windowSize(3)
+        .accumulator(function(values) {
+            if (values[0] === undefined) {
+                return undefined;
+            }
+            return d3.mean(values);
+        });
+
+    var stochastic = function(data) {
+        var kValues = kWindow(data);
+        var dValues = dWindow(kValues);
+        return kValues.map(function(k, i) {
```

Comment:
I'm not sure what kind of reception that will get. zip already has a pretty well defined behavior (not just in d3) of returning an array of arrays and you can achieve the same thing (albeit with an extra iteration) without much effort -
```js
d3.zip(kValues, dValues).map(([k, d]) => { k, d })
``` 

If `kValues` and `dValues` are known to have the same size, I'm not sure what advantage we get from using zip? If it isn't giving us anything, I actually prefer the existing implementation (I might quibble over inlining `d` but meh).